mindmeld/converter/dialogflow.py

```diff
@@ -0,0 +1,375 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2015 Cisco Systems, Inc. and others.  All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This module contains the DialogFlowConverter class used to convert DialogFlow projects
+into Mindmeld projects"""
+
+import re
+import os
+import json
+import logging
+from sklearn.model_selection import train_test_split
+
+from mindmeld.converter.converter import Converter
+
+logger = logging.getLogger(__name__)
+
+
+class DialogFlowConverter(Converter):
+    """The class is a sub class of the abstract Converter class. This class
+    contains the methods required to convert a Dialogflow project into a MindMeld project
+    """
+    sys_entity_map = {'@sys.date-time': 'sys_interval',
```

Comment:
Mention the entity keys that are supported in DF and not in MindMeld as a `#ToDo` item

Reply:
Ok, I added a list of system entities support in DF and not Mindmeld with a comment above.